project/CodeCoverageSettings.scala

```diff
@@ -33,7 +33,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimum := 95,
+    ScoverageKeys.coverageMinimum := 75,
```

Comment:
Is this a temporary value?

Reply:
yeah

Reply:
can you add a comment to say it needs to go back